calendars/contributor-experience.yaml

```diff
@@ -0,0 +1,22 @@
+name: Contributor Experience Project Community Calendar
+events:
+  - summary: Contributor Experience Project Community Call
+    description: |
+      A bi-weekly meeting to discuss everything related to contributor experience. Everyone is welcome to attend and contribute to a conversation.
+    begin: 2022-10-31 09:00:00 +00:00
```

Comment:
Can we get rid of this entry now?

Reply:
@stefanv Could you please clarify what entry you are referring to?